sentry-rails/lib/sentry/rails/action_cable.rb

```diff
@@ -0,0 +1,72 @@
+module Sentry
+  module Rails
+    module ActionCable
+      class ErrorHandler
+        ACTION_CABLE_NAME = 'ActionCable'
+
+        def self.capture(env, transaction_name:, extra_context: nil, &block)
+          Sentry.with_scope do |scope|
+            scope.set_rack_env(env)
+            scope.set_extras(action_cable: extra_context) if extra_context
+            scope.set_transaction_name("#{ACTION_CABLE_NAME}/#{transaction_name}")
```

Comment:
since we already prefix it `transaction_name` with the channel's name, like `AppearanceChannel#subscribed`, I don't think the additional `ActionCable` namespace is necessary.